listenbrainz/webserver/templates/user/import.html

```diff
@@ -58,4 +58,16 @@ <h3>Import a Last.fm backup file</h3>
       return false;
     }
   </script>
+  <script type="text/javascript">
```

Comment:
Should probably add a one line comment explaining the need for this script.

Reply:
:+1: